src/test/java/org/cactoos/io/GzipInputTest.java

```diff
@@ -43,13 +43,11 @@
     public void readFromGzipInput() throws Exception {
         final byte[] bytes = {
             (byte) GZIPInputStream.GZIP_MAGIC,
-            // @checkstyle MagicNumberCheck (1 line)
-            (byte) (GZIPInputStream.GZIP_MAGIC >> 8), (byte) 0x08,
-            (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00,
-            (byte) 0x00, (byte) 0x00, (byte) 0xF3, (byte) 0x48, (byte) 0xCD,
-            (byte) 0xC9, (byte) 0xC9, (byte) 0x57, (byte) 0x04, (byte) 0x00,
-            (byte) 0x56, (byte) 0xCC, (byte) 0x2A, (byte) 0x9D, (byte) 0x06,
-            (byte) 0x00, (byte) 0x00, (byte) 0x00,
+            -117, 8, 0, 0, 0,
```

Comment:
@Umbrah in the case of `GzipInput`, the problem I see is that these are a bunch of magic numbers that the reader needs to either trust that they represent "Hello!", or put in some effort to verify it themselves on paper or somewhere else.

The readability of this test can be improved by using the Java SDK to serialize "Hello!" to an array of gzipped bytes. Then, these same bytes are read back with `GzipInput` and the contents have to be the same "Hello!" string. The benefits here are that a) there are no magic numbers, and b) we set up a nice BUILD-OPERATE-CHECK pattern where there is a clear correlation between the BUILD and the CHECK.